src/test/java/com/wikia/webdriver/common/core/ArticleContent.java

```diff
@@ -60,12 +65,15 @@ public static void push(String text, String articleTitle) {
 
       PageObjectLogging.log("CONTENT PUSH", "Content pushed to article: " + baseURL + articleTitle,
           true);
-    } catch (ClientProtocolException | URIException e) {
+    } catch (ClientProtocolException e) {
       PageObjectLogging.log("EXCEPTION", e.toString(), false);
       throw new WebDriverException("Problem with content pushing");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Problem with content pushing" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)